app_test.go

```diff
@@ -2265,3 +2265,239 @@ func TestWhenExitSubCommandWithCodeThenAppQuitUnexpectedly(t *testing.T) {
 		t.Errorf("exitCodeFromOsExiter valeu should be %v, but its value is %v", testCode, exitCodeFromExitErrHandler)
 	}
 }
+
+func TestDuplicateCommandName(t *testing.T) {
+	duplicateCommandName := "hello"
+	os.Args = []string{"say", duplicateCommandName}
+	app := NewApp()
+	app.Name = "say"
+	app.Commands = []Command{
+		{
+			Name: duplicateCommandName,
+			Action: func(c *Context) error {
+				fmt.Println("Hello")
```

Comment:
⭐⭐⭐ _(must change)_ on this line and the other similar lines, remove the `print`s. We don't want to output a ton of non-actionable test messages when running tests.